R/zchunk_LA123.electricity.R

```diff
@@ -29,26 +29,118 @@ module_energy_LA123.electricity_DISABLED <- function(command, ...) {
 
     all_data <- list(...)[[1]]
 
+
+
     # Load required inputs
     enduse_fuel_aggregation <- get_data(all_data, "energy/enduse_fuel_aggregation")
     A23.chp_elecratio <- get_data(all_data, "energy/A23.chp_elecratio")
-    L1011.en_bal_EJ_R_Si_Fi_Yh <- get_data(all_data, "L1011.en_bal_EJ_R_Si_Fi_Yh")
+    L1011.en_bal_EJ_R_Si_Fi_Yh <- get_data(all_data, "energy/L1011.en_bal_EJ_R_Si_Fi_Yh")
```

Comment:
If this isn't available yet from the new data system, this file should go into `temp-data-inject/`. You should then immediately transform it to long form, e.g.:

```R
get_data(all_data, "temp-data-inject/L1011.en_bal_EJ_R_Si_Fi_Yh") %>%
  gather(year, value, ...) %>%
  mutate(year = as.integer(substr(year, 2, 5)) ->
  L1011.en_bal_EJ_R_Si_Fi_Yh
```